### PR TITLE
Webtest added for Cachecontrol form

### PR DESCRIPTION
**Pull request by neetumorwani**

### CODE REVIEW

**Comment by neetumorwani** [comment]
@cam8001 Did you get a chance to review the PR?

File: src/Tests/AkamaiCacheControlFormTest.php [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      'administer akamai',
+      'purge akamai cache',
+    ));
+    $this->drupalLogin($this->privilegedUser);
+    $this->drupalCreateContentType(['type' => 'article']);
+    $this->node = $this->drupalCreateNode(['type' => 'article']);
+  }
+
+  /**
+   * Tests that manual Akamai Cache Clear page.
```
Comment (cam8001) [inline src/Tests/AkamaiCacheControlFormTest.php]: Suggested comment - "Tests manual purging via Akamai Cache Clear form."

File: src/Tests/AkamaiCacheControlFormTest.php [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   */
+  public function testValidUrlPurging() {
+
+    $edit['paths'] = 'node/1';
+    $edit['domain_override'] = 'staging';
+    $edit['action'] = 'invalidate';
+
+    $this->drupalPostForm('admin/config/development/performance/akamai-cache-clear', $edit, t('Start Refreshing Content'));
+    $this->assertText(t('Requested invalidate of the following URLs: /node/1'), t('node/1 URLs purged'));
```
Comment (cam8001) [inline src/Tests/AkamaiCacheControlFormTest.php:62]: Should we add a negative test as well? i.e - adding a path that we know doesn't exist and making sure that the form rejects it.

File: src/Tests/AkamaiCacheControlFormTest.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    $this->drupalLogin($this->privilegedUser);
+    $this->drupalCreateContentType(['type' => 'article']);
+    $this->node = $this->drupalCreateNode(['type' => 'article']);
+  }
+
+  /**
+   * Tests that manual Akamai Cache Clear page.
+   */
+  public function testValidUrlPurging() {
+
```
Comment (cam8001) [inline src/Tests/AkamaiCacheControlFormTest.php]: I don't think we need these blank lines after each function declaration.

**Comment by cam8001** [comment]
Nice, thanks! I added some nitpicky comments. I've also finally got travis working, so we should be able to see test results (including code style stuff) on PR pushes.

https://travis-ci.org/d8-contrib-modules/akamai

File: src/Tests/AkamaiCacheControlFormTest.php (hunk 2 above)
Comment (neetumorwani) [inline src/Tests/AkamaiCacheControlFormTest.php:62]: We are checking for Invalid URL in the next method of the class-  testInvalidUrlPurging() .

**Comment by neetumorwani** [comment]
PR updated with all the fixes. All the tests pass on travis as well.
[https://travis-ci.org/d8-contrib-modules/akamai](https://travis-ci.org/d8-contrib-modules/akamai)
@cam8001  Can you please review and merge the PR?

File: src/Tests/AkamaiCacheControlFormTest.php (hunk 2 above)
Comment (naveenvalecha) [inline src/Tests/AkamaiCacheControlFormTest.php:62]: IMO merge both methods into single one and first do postitive testing and then negative testing in this way test don't need to instantiate the environment again.

Comment (cam8001) [inline src/Tests/AkamaiCacheControlFormTest.php:62]: +1

Comment (neetumorwani) [inline src/Tests/AkamaiCacheControlFormTest.php:62]: Cool. I can do that.

**Comment by neetumorwani** [comment]
PR updated.

**Comment by cam8001** [comment]
Well look at that Travis came to the party.

**Pull request merged**